common/views/components/SearchFilters/SearchFilters.Colors.tsx

```diff
@@ -0,0 +1,25 @@
+import dynamic from 'next/dynamic';
+import { ColorFilter as ColorFilterType } from '@weco/common/services/catalogue/filters';
+
+const PaletteColorPicker = dynamic(
+  import('@weco/common/views/components/PaletteColorPicker/PaletteColorPicker')
+);
```

Comment:
Why does this need to be a dynamic import?

Reply:
I mostly copy/pasted, I can test without and see if it's required (@AMBacelar might have a faster answer in the meantime).

Reply:
And actually looking at this again, it's just redeclaring the same component. Might as well straight up use `PaletteColorPicker`. I'll do that next.

Reply:
I've done that change now [Remove Color component and use PaletteColorPicker directly](https://github.com/wellcomecollection/wellcomecollection.org/pull/9245/commits/b08ed24a4c5780acfc0d4f0a79e58fdf704e2dac)

@AMBacelar do let me know if the dynamic import was because of a bug you came across! I tested most scenarios again and it looked OK.